code/game/objects/effects/aliens.dm

```diff
@@ -183,7 +183,6 @@
 			Life()
 
 /obj/structure/alien/weeds/Destroy()
-	loc = null

```

Comment:
It was to update the weed overlays back when it used the stupid edging stuff and I forgot to remove it, so it'd move to null, get destroyed, and not update adjacent smoothing because it doesn't have adjacentcies, it's in null